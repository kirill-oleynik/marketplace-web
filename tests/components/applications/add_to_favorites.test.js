const React = require('react');
const { shallow } = require('enzyme');
const toJSON = require('enzyme-to-json').default;
const {
  AddToFavorites
} = require('../../../app/components/applications/add_to_favorites');

describe('#render', () => {
  test('it renders correctly', () => {
    const component = shallow(
      <AddToFavorites
        t={() => {}}
        disabled
        onClick={() => {}}
      />
    );

    expect(
      toJSON(component)
    ).toMatchSnapshot();
  });
});
