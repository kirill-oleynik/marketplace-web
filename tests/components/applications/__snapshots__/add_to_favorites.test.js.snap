// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`#render it renders correctly 1`] = `
<button
  className="favourites-add"
  disabled={true}
  onClick={[Function]}
  type="button"
>
  <i
    className="icon icon-heart font-16 mr-10 favourites-heart__icon"
  />
  <span
    className="favourites-add__text"
  />
</button>
`;
